.travis.yml

```diff
@@ -33,11 +34,5 @@ matrix:
         - docker exec analytics_api_testing /edx/app/analytics_api/analytics_api/.travis/run_coverage.sh
         - codecov --disable pycov
     - env:
-        TESTNAME=test-python-django20
-        TARGETS="DJANGO_VERSION=django20 test"
-    - env:
-        TESTNAME=test-python-django21
-        TARGETS="DJANGO_VERSION=django21 test"
-    - env:
-        TESTNAME=test-python-django111
-        TARGETS="DJANGO_VERSION=django111 test"
+        TESTNAME=test-python-django22
+        TARGETS="DJANGO_VERSION=django22 test"
```

Comment:
Hi, I think that you should put in TARGETS="PYTHON_ENV=py35" in order to use that version because all of these tests run on the container and not in the travis environment.

Reply:
Hi @morenol, thanks for the advice. I've made the changes according to your suggestion. Could you have a look again and see if I am missing anything else? Thanks.